cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -34,12 +34,16 @@
   @Override
   public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
     if (t instanceof BadRequestException) {
+      logWarnings(request, t);
```

Comment:
Shouldn't log warning in all these cases are they are error caused by user, meaning someone can have a script that keep performing invalid action to flood the server log.